### DISCUSSION TITLE
Example won't build or Run

### DISCUSSION THREAD

**Discussion by NeighborhoodNerd** [AvaloniaInside/Shell#57] [Q&A] [answered]
I'd really like to switch from MAUI to Avalonia. My app uses a Flyout control and this project seems to be the best way to achieve that. I've pulled the example down and I get 99+ errors. Please help.

![image](https://github.com/user-attachments/assets/f2d523f6-b9a7-491a-ad7d-dc105a76fac0)

**Comment by OmidID** [comment]
Hello @nathenxbrewer 

The main and develop branch are clean and no error. What I see with that much of errors. you need to do restore, rebuild and be sure you have dotnet 8 install. 
Please check your computer, environment and sdks. 
I could say this is not issue of this project.

I suggesting to use branch `main` that you will be sure it is same as production.

Thanks.

**Comment by NeighborhoodNerd** [comment]
@OmidID I've tried all of this. Nothing changes. 
![image](https://github.com/user-attachments/assets/2d7bcc16-20ea-473a-8934-d376f886b14a)

**Comment by OmidID** [comment]
All the error I can see in your screenshot it related to MSBuild. 

For something something very wrong with your SDK or packages. 

In the command line manually run dotnet restore to see what happening.

**Comment by NeighborhoodNerd** [comment]
![image](https://github.com/user-attachments/assets/2bcb0dde-23d4-4a0b-9524-5dd25e019d3d)


To clarify, I am on a MacBook, using Rider. Arm64

**Comment by NeighborhoodNerd** [comment, marked as answer]
@OmidID It's working now. Had to re-launch Rider multiple times, do a few Cleans, Rebuilds, dotnet restore, but looks like the final dotnet restore finally got it all working. Thanks

**Comment by OmidID** [comment]
So Classic. I move this issue to discussion

**Comment by NeighborhoodNerd** [comment]
@OmidID After getting it to run, iOS is looking very strange. 

![image](https://github.com/user-attachments/assets/98dfeab2-e48a-4720-89c0-b0eed1602197)


![image](https://github.com/user-attachments/assets/09294a5f-c1d5-480b-874e-98a0b8504317)


When I run it from the 45-keep-transition-animation-for-each-navigate, it looks fine.

**Reply by OmidID** [reply]
This is already fixed and exists in `develop` and will release in next version

**Reply by OmidID** [reply]
https://github.com/AvaloniaInside/Shell/issues/42
